Resources/Locale/en-US/voting/vote-commands.ftl

```diff
@@ -12,8 +12,10 @@ cmd-createvote-arg-vote-type = <vote type>
 
 cmd-customvote-desc = Creates a custom vote
 cmd-customvote-help = Usage: customvote <title> <option1> <option2> [option3...]
-cmd-customvote-on-finished-tie = Tie between {$ties}!
-cmd-customvote-on-finished-win = {$winner} wins!
+cmd-customvote-on-finished-tie = Custom vote have been made: {$title}
+                                 Tie between {$ties}!
+cmd-customvote-on-finished-win = Custom vote have been made: {$title}
+                                 {$winner} wins!
```

Comment:
This definitely reads weird. Make it like "the vote '{title}' has finished: {bla bla} wins!" maybe?